cmd/utils/flags.go

```diff
@@ -1749,3 +1852,14 @@ func getNetworkId(ctx *cli.Context) (uint64, bool) {
 		return params.CypressNetworkId, false
 	}
 }
+
+func getFlagName(ctx *cli.Context, flag cli.Flag, legacyFlag cli.Flag, deprecateVersion string) string {
```

Comment:
@hyochan-brown It seems that idential `getFlagName` is defined in cmd/kbn/config.go.. Could you check this?

Reply:
It was just made to remove the duplicate code, I thought it didn't have to be open in the external package.
So I added it on both flags.go / config.go... 
Is it better to open it on the external package to remove deduplication?